### PR TITLE
Confirm privacy policy minimum age at registration

### PR DESCRIPTION
**Pull request by Tirokk**
> [ogerly](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-10-15T17:23:57Z" title="Tuesday, October 15th 2019, 7:23:57 pm +02:00">Oct 15, 2019</time>_
_Merged <time datetime="2019-10-15T21:41:01Z" title="Tuesday, October 15th 2019, 11:41:01 pm +02:00">Oct 15, 2019</time>_
---

## 🍰 Pullrequest
Confirm data privacy policy + minimum age at registration

  
ezgif com-video-to-gif

### CODE REVIEW

File: webapp/components/Registration/CreateUserAccount.vue [hunk 1]
```diff
@@ -74,12 +74,24 @@
...
+            <input id="checkbox1" type="checkbox" v-model="dataPrivacy" :checked="dataPrivacy" />
+            <span v-html="$t('components.registration.signup.form.data-privacy')"></span>
+          </label>
+        </p>
+        <p>
+          <label>
+            <input id="checkbox2" type="checkbox" v-model="minimumAge" :checked="minimumAge" />
+            <span v-html="$t('components.registration.signup.form.minimum-age')"></span>
+          </label>
+        </p>
```
Comment (Tirokk) [inline webapp/components/Registration/CreateUserAccount.vue:88]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-15T21:38:05Z" title="Tuesday, October 15th 2019, 11:38:05 pm +02:00">Oct 15, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -74,12 +74,24 @@
             v-html="$t('termsAndConditions.termsAndConditionsConfirmed')"
           ></label>
         </ds-text>
+        <p>
+          <label>
+            <input id="checkbox1" type="checkbox" v-model="dataPrivacy" :checked="dataPrivacy" />
+            <span v-html="$t('components.registration.signup.form.data-privacy')"></span>
+          </label>
+        </p>
+        <p>
+          <label>
+            <input id="checkbox2" type="checkbox" v-model="minimumAge" :checked="minimumAge" />
+            <span v-html="$t('components.registration.signup.form.minimum-age')"></span>
+          </label>
+        </p>
```


We desparately need the checkbox form input.

**Comment by Tirokk** [comment]
> [datenbrei](https://github.com/datenbrei) **Authored by [datenbrei](https://github.com/datenbrei)**
_<time datetime="2019-10-16T07:25:40Z" title="Wednesday, October 16th 2019, 9:25:40 am +02:00">Oct 16, 2019</time>_

---


Yes, looks good.

**Review by Mogge — APPROVED** [review]
I thought we talked about being able to have one checkbox for both `ToC` and `Data Privacy`, but I don't think it's a reason to block this from getting merged...

**Review by Mogge — APPROVED** [review]
looks good to me

Looks good to me! :+1: 

Of course we should refactor those `checkbox1`, checkbox2`, `checkbox3` with something more telling.

**Pull request closed without merging**